features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Configuration/RuntimeConfigurationContext.jsx

```diff
@@ -0,0 +1,6 @@
+import React, { useContext } from 'react';
+
+const RuntimeConfigurationContext = React.createContext({ });
```

Comment:
Did you define a context to avoid passing `setSaveDisable` across `RuntimeConfig -> APISecurity -> ApplicationLevel -> AuthorizationHeader` (props drill down) but `configDispatcher` is already there in the `AuthorizationHeader` so let's try to utilize that :)

Reply:
Yes, that was the idea behind introducing the new context. Thanks for the suggestion, I have fixed this with https://github.com/wso2/carbon-apimgt/pull/10192/commits/82a33e7c752652ee92ae647ace14fe9c218e6650